package-lock.json

```diff
@@ -3567,6 +3568,43 @@
       "resolved": "https://registry.npmjs.org/spdx-license-ids/-/spdx-license-ids-3.0.11.tgz",
       "integrity": "sha512-Ctl2BrFiM0X3MANYgj3CkygxhRmr9mi6xhejbdO960nF6EDJApTYpn0BQnDKlnNBULKiCN1n3w9EBkHK8ZWg+g=="
     },
+    "node_modules/spec-up": {
```

Comment:
This as well.

Reply:
ok - removed these (unnecessary) changes to package-lock.json.